onnx/defs/rnn/defs.cc

```diff
@@ -235,6 +235,7 @@ Equations (Default: f=Sigmoid, g=Tanh):
 
 
 OPERATOR_SCHEMA(LSTM)
+    .SinceVersion(2)
```

Comment:
When checking in this change, kindly remember to bump the global version in schema.h accordingly please. The version here may be updated too if the global version is changed by other users. For example, PR #390 .

Reply:
Yep, please do this